.github/workflows/mockito.yaml

```diff
@@ -0,0 +1,109 @@
+name: package:mockito
```

Comment:
Can we migrated this to use package:mono_repo (add a `mono_repo.yaml`)?

It won't be a good situation to have multiple packages in the same repo managing their CI in different ways.

Reply:
As long as mono_repo doesn't break in this situation, I'm not very worried about mixing the types of workflows we use in a single repo.

What types of problems are you worried might come up?

Reply:
If we switch, I would prefer switching to `mono_repo` in a later PR. I have no experience rewriting CI files into the `mono_pkg.yaml`, and would prefer landing this merge rather quickly before anyone updates the original repository.

(To be had in a separate discussion, but I am not a big fan of `package:mono_repo`. It introduces complexity in a new file format with `mono_pkg.yaml`, generates large hard to read `dart.yml` files, and the status checks at the end of PRs do not look nice. In my experience, the CI files need little refactorings, so I don't see the advantage of generating them.)

Reply:
> What types of problems are you worried might come up?

It is just going to be a maintenance headache having to deal with two different systems. I don't want to manage manual github workflows.

Reply:
I am fine with doing this in a followup